package.json

```diff
@@ -12,7 +12,7 @@
     "gatsby-image": "2.4.9",
     "gatsby-plugin-google-tagmanager": "2.3.7",
     "gatsby-plugin-manifest": "2.4.14",
-    "gatsby-plugin-offline": "2.2.10",
+    "gatsby-plugin-offline": "3.2.28",
```

Comment:
Potentially breaking change. We ought to test on the preview branch before going ahead with this.

Reply:
This plugin is unused in this repo https://github.com/neontribe/www/blob/dc9cb589ea5d8d51f130dbe2d10cb82bd4784fd4/gatsby-config.js#L80-L86

I think we can merge it safely, or remove the dep completely.